romm/umbrel-app.yml

```diff
@@ -0,0 +1,26 @@
+manifestVersion: 1
+id: room
+category: media
+name: RomM
+icon: https://raw.githubusercontent.com/zurdi15/romm/a700aa98429b0ec96ec8d9d32ffa01c989d3569f/romm.svg
+version: "1.10"
+tagline: Rom Manager.
+description: >-
+  RomM (Rom Manager) is a web based retro roms manager integrated with IGDB.
+releaseNotes: >-
```

Comment:
Since this is the first version you can remove the releaseNotes and replace it with:

```
releaseNotes: ""
```